mlflow/models/evaluation/base.py

```diff
@@ -212,6 +382,21 @@ def eval_fn(
         genai_metric_args: (Optional) A dictionary containing arguments specified by users
             when calling make_genai_metric or make_genai_metric_from_prompt. Those args
             are persisted so that we can deserialize the same metric object later.
+        standard_signature: (Optional) Whether the metric follows a strict signature, if True then
+            the eval_fn must follow below signature:
+
+                .. code-block:: python
+
+                    def eval_fn(
+                        predictions: "pd.Series",
+                        metrics: Dict[str, MetricValue],
+                        inputs: "pd.Series",
+                        *args,
+                    ) -> MetricValue:
+                        pass
+
+            When generating a metric from `make_genai_metric`, this should be set to True.
+            Default to False.
```

Comment:
`standard_signature` doesn't sound like a flag. Could we consider renaming it to something more indicative of its purpose, such as `enforce_standard_signature` or `require_strict_signature` for clarity?

Reply:
I'd prefer `require_strict_signature` then!